.github/workflows/ci.yaml

```
@@ -50,3 +50,53 @@ jobs:
           labels: ${{ steps.docker_meta.outputs.labels }}
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+          outputs: type=docker,dest=/tmp/workload-standard-${{ github.sha }}.tar
+      - name: Upload artifact
```

```suggestion
      - name: Upload artefact
```

nitpick: we use British spelling internally.

---

hmm, that will cause confusion since most of community documents or guideline are using this world `artifact`.